### DISCUSSION TITLE
Command for ALL NOTES OFF

### DISCUSSION THREAD

**Discussion by TomC53** [FortySevenEffects/arduino_midi_library#207] [Q&A] [answered]
I looked for this in sample code and couldn't find it,  and also tried figure this out from the documentation but cannot.  I just need to know how to send the proper command to turn all notes off.  Thank you in advance!

**Comment by franky47** [comment, marked as answer]
The [MIDI specification](http://www.somascape.org/midi/tech/spec.html#chanmodemsgs) specifies CC 123 for AllNotesOff, and [there is an alias for it](https://github.com/FortySevenEffects/arduino_midi_library/blob/master/src/midi_Defs.h#L211) so this should do the trick:
```c++
MIDI.sendControlChange(midi::AllNotesOff, 0, yourChannelHere);
```

**Reply by TomC53** [reply]
Yes that works.  Thank you very much!
